tasks/configure.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: Deploy config file
+  become: yes
+  become_user: "{{ sansible_prometheus_alertmanager.user }}"
+  template:
```

Comment:
Can we remove the commented lines if they are not needed?

Reply:
done